backend-project/small_eod/files/serializers.py

```diff
@@ -6,7 +6,18 @@ class FileSerializer(serializers.ModelSerializer):
     class Meta:
         model = File
         fields = [
-            "letter",
+            "id",
+            "path",
+            "name",
+        ]
+
+
+class FileRelatedSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = File
+        fields = [
+            "id",
             "path",
             "name",
+            "letter",
```

Comment:
Dlaczego tego potrzebujesz? Przez to potem "hakujesz" w ```CreateFileView```, a przecież wartość ```letterId``` chcesz zawsze przy zapisie przekazać w path (nie w body), a przy odczycie nie warto zwracać wartości w body, którą podałeś w path. 

Ewentualnie oznaczyć pole jako ```read_only``` i zaktualizować dokumentacje. To też jest dla mnie całkiem sensowne, zwłaszcza jeżeli będzie to spójne z sposobem zwracania innych zagnieżdżonych obiektów.

Reply:
Problem pojawiał się przy zapisie poprzez serializer. Brakowało mi pomysłów jak sobie z tym z  w ładny sposób poradzić, dlatego zdecydowałem się na nowy serializer, który umożliwił mi zapis ale wprowadził niezgodność z dokumentacją. Skłaniałbym się ku read_only + poprawa dokumentacji.